Classes/Repository/RepositoryViewController.swift

```diff
@@ -12,18 +12,27 @@ import Pageboy
 import TUSafariActivity
 import SafariServices
 
+enum RepositoryPageIndex: Int {
+    case overview    = 0
+    case issues      = 1
```

Comment:
Issues is not always available (see the `controllers` array in the init) so these numbers will immediately be out in such a repository

Reply:
🤒  forgot about that totally! Gonna make some change

Reply:
I'm stuck 😢  @Sherlouk do you have any way to tackle this w/ or w/out `enum`?

Reply:
RepositoryPageIndex could have a fileprivate var which is the view controller type (not an initialised version) e.g.  `RepositoryIssuesViewController.self`. Then `defaultPage(for:)` is the index of the first item in the `controllers` array where item is of type `defaultPageIndex.viewController`

Reply:
but we use `RepositoryIssuesViewController` for both issue and pull request. So when we want to default index to be a pull request it will fail. isn't?

Reply:
Fair point, I didn't realise that - Maybe a protocol then which specifies the PageType so you can iterate through the controllers array and find the first item that matches PageType with the enum you pass through for default.